### ISSUE TITLE
UI problem after resume from sleeping

### ISSUE THREAD

**Issue by Tevic** [OpenVPN/openvpn-gui#64]
```
OpenVPN: OpenVPN 2.3.11-- released on 2016.05.10
OS: Windows 10.0.10586(x64)
```

![20160719102152](https://cloud.githubusercontent.com/assets/958787/16936620/adbf3f7e-4d9a-11e6-935d-e48a2b119124.png)

As shown above, the icon missing after resume from sleeping.

**Comment by mattock** [comment]
Does this also happen with OpenVPN GUI bundled with OpenVPN 2.4.0?

**Comment by Tevic** [comment]
It seems the bug is fixed in 2.4.0
